python/lsst/pipe/base/execution_reports.py

```diff
@@ -235,7 +248,17 @@ def to_summary_dict(self, butler: Butler, do_store_logs: bool = True) -> dict[st
                     quantum_info["error"] = [
                         record.message for record in log if record.levelno >= logging.ERROR
                     ]
-            failed_quanta[str(node_id)] = quantum_info
+            if human_readable:
+                failed_quanta["data_id"] = data_ids
```

Comment:
This ought to be easy enough to fix; you could just repeat whatever test calls this function and explicitly set human_readable to True in one case and not in the other.

Reply:
Let's see if that works! I repeated the same tests (which should work, because the only difference between these reports is in the `failed_quanta` section which we don't introspect into much). I'll merge if my attempt at following all your suggestions pass Jenkins.